server/docker-compose.yml

```diff
@@ -34,6 +34,6 @@ services:
       PGADMIN_DEFAULT_EMAIL: spope@blockchains.com
       PGADMIN_DEFAULT_PASSWORD: password
     ports:
-      - "5000:80"
```

Comment:
if you have airdrop running on mac it interrupts this port, so I changed to 4000